os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -178,6 +178,43 @@ static inline void up_registerdump(void)
 #define up_registerdump()
 #endif
 
+/****************************************************************************
+ * Name: up_taskdump
+ ****************************************************************************/
+
+#ifdef CONFIG_STACK_COLORATION
+static void up_taskdump(FAR struct tcb_s *tcb, FAR void *arg)
```

Comment:
Why is it taskdump? not stackinfodump?

Reply:
Please forget my previous comment. But anyway ```taskdump``` is not an approprietary name I think. 
1. change a name to ```up_tasklist```?
2. add more information like tcb_s address and content because name is dump.

Reply:
It's copy of `up_taskdump` from below.  Do I need to change?
https://github.com/Samsung/TizenRT/blob/c356fc184471438d8f8b2b4d5b84fee396b814ad/os/arch/arm/src/armv7-r/arm_assert.c#L474

Reply:
I don't think that is a good name. But it comes from Nuttx. Let's keep original name for maintainance.